frontend/src/routes/(authenticated)/project/[project_code]/+page.svelte

```diff
@@ -242,20 +245,39 @@
               <div class="prose">
                 <h3>{$t('project_page.get_project.instructions_header', {type: project.type, mode: 'normal'})}</h3>
```

Comment:
We should also change this text as well as the button text on line 240.
How about:
- "Set up project" for the button (I spent about 3 minutes thinking about it 😜)
- "To set up this project with {type}" (you can copy the type part from the current message) for the instructions header

```suggestion
                <h3>{$t('project_page.get_project.instructions_header', {type: project.type, mode: 'normal'})}</h3>
```

Reply:
I think "Set up project" when it's non-empty doesn't communicate the same thing as "Get project"; people might think it means that they need to take more steps on Language Depot / LexBox when in fact they just need to "Get project". I'll change the button text, but only when the project is empty.

Reply:
Done in commit 292a1af7.

Reply:
Yeah, I only meant that we should change it for the empty case 👍